app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/LoginPromtCard.java

```diff
@@ -0,0 +1,61 @@
+package de.tum.in.tumcampusapp.component.ui.onboarding;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.ui.overview.CardManager;
+import de.tum.in.tumcampusapp.component.ui.overview.card.Card;
+import de.tum.in.tumcampusapp.utils.Const;
+import de.tum.in.tumcampusapp.utils.Utils;
+
+/**
+ * Card that describes how to dismiss a card
```

Comment:
Nope, that's not what that card does :)

Maybe add the reasoning from #742 here?